src/main/java/games/strategy/engine/framework/GameRunner.java

```diff
@@ -110,21 +125,31 @@
 
 
 
+
   /**
    * Launches the "main" TripleA gui enabled game client.
    * No args will launch a client, additional args can be supplied to specify additional behavior.
    * Warning: game engine code invokes this method to spawn new game clients.
    */
   public static void main(final String[] args) {
-    ErrorConsole.getConsole();
     if (!ArgParser.handleCommandLineArgs(args, COMMAND_LINE_ARGS)) {
       usage();
       return;
     }
+    SwingUtilities.invokeLater(LookAndFeel::setupLookAndFeel);
+    ErrorConsole.getConsole();
     // do after we handle command line args
     Memory.checkForMemoryXMX();
 
-    SwingUtilities.invokeLater(LookAndFeel::setupLookAndFeel);
+    gameSelectorModel = new GameSelectorModel();
+    setupPanelModel = new SetupPanelModel(gameSelectorModel);
```

Comment:
Both `gameSelectorModel` and `setupPanelModel` are accessed by multiple threads.  Instead of initializing both `gameSelectorModel` and `setupPanelModel` here and worrying about making them `volatile`, it looks like they can be made `final` and initialized via field initializers or in the class initializer.  For example:

```java
private static final GameSelectorModel gameSelectorModel = new GameSelectorModel();
private static final SetupPanelModel setupPanelModel = new SetupPanelModel(gameSelectorModel);
```